exercises/custom-set/example.ml

```diff
@@ -1,3 +1,5 @@
+open Core.Std
+
```

Comment:
Bu then people will have to learn to code against 2 APIs as Core is somewhat different to the standard library.